apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -1,92 +1,142 @@
+
         const objs = await Organization.getAll(client, {})
-        expect(objs.length).toBeGreaterThan(0)
+        expect(objs.length).toBe(1)
+        expect(objs[0].id).toEqual(client.organization.id)
+    })
+
+    test('anonymous: read Organization', async () => {
+        const client = await makeClient()
+
+        try {
+            await Organization.getAll(client)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['objs'],
+            })
+            expect(e.data).toEqual({ 'objs': null })
+        }
+    })
+
+    // Test case, where user can update organization are in OrganizationEmployeeRole.test.js
+    test('user: update Organization — cannot', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [objCreated] = await createTestOrganization(admin)
+
+        const client = await makeClient()
+        const payload = {
+            name: faker.company.companyName(),
+        }
+
+        try {
+            await updateTestOrganization(client, objCreated.id, payload)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['obj'],
+            })
+            expect(e.data).toEqual({ 'obj': null })
+        }
     })
 
-    test('user: allow to count', async () => {
+    test('anonymous: update Organization', async () => {
         const admin = await makeLoggedInAdminClient()
-        await createOrganization(admin)
-        const [, userAttrs] = await createTestUser(admin)
-        const client = await makeLoggedInClient(userAttrs)
-        const count = await Organization.count(client, {})
-        expect(count).toBeGreaterThan(0)
+        const [objCreated] = await createTestOrganization(admin)
+
+        const client = await makeClient()
+        try {
+            await updateTestOrganization(client, objCreated.id)
+        } catch (e) {
+            expect(e.errors[0]).toMatchObject({
+                'message': 'You do not have access to this resource',
+                'name': 'AccessDeniedError',
+                'path': ['obj'],
+            })
+            expect(e.data).toEqual({ 'obj': null })
+        }
     })
 
-    test('default status transitions is defined', async () => {
-        const { organization } = await makeClientWithRegisteredOrganization()
+    test('user: delete Organization', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [objCreated] = await createTestOrganization(admin)
 
-        expect(organization.statusTransitions).toBeDefined()
-        expect(organization.defaultEmployeeRoleStatusTransitions).toBeDefined()
+        const client = await makeClient()
```

Comment:
user client != makeClient! check another tests

Reply:
Fixed in `3eed848b`